src/redisearch_api.c

```diff
@@ -510,6 +518,12 @@ void RediSearch_IndexOptionsSetGetValueCallback(RSIndexOptions* options, RSGetVa
   options->gvcbData = ctx;
 }
 
+void RediSearch_IndexOptionsSetStopwords(RSIndexOptions* opts, const char **stopwords, int stopwordsLen) {
+  opts->stopwords = stopwords;
```

Comment:
I think it a good idea to perform a full copy of this array, we have no idea if the given argument is stack-allocated or heap-allocated. It's not something that happened a lot and I think it's OK to copy it.

Reply:
And then you also need to free it on RediSearch_FreeIndexOptions

Reply:
@MeirShpilraien  We don't need these strings. We just use them to populate a trie. This is why we use `const`.

Reply:
Who promises those strings will not be free by the time you create the index? Who promises the user will not create one index option and will use it again and again to create indexes?

Reply:
Done